stories.html

```diff
@@ -0,0 +1,90 @@
+<html>
+   <head>
+            <title>Latest Stories</title>
+            <style>
+                body{
+                    margin-top: 0px;
+                     margin-right: 0px;
+                     margin-bottom: 0px;
+                    margin-left: 0px;
+                }
+                ul{
+                    padding-inline-start: 1px;
+                    
+                }
+                ul>li{
+                    list-style-type: none;
+                    float:left;
+                    padding-right: 10px;
+                    
+                   
+                }
+                p{
+                    font-weight: bold;
+                    font-size:18px;
+                    font-family: 'Times New Roman';
+                    text-align: justify;
+                    padding-top: 0px;
+                   
+                }
+                
+                .vertical { 
+			        border-left: 1px solid gray; 
+			        height: 325px; 
+                    padding-right: 10px;
+			       	
+                } 
+                figure-caption{
+                    font-family:Arial, Helvetica, sans-serif;
+                    font-size: 12px;
+                    color: gray;
+                }
+            
+            </style>
+    </head>
+    <body>
+        <header>
+            <h2>Latest Stories</h2>
+        </header>
+        
+        <article>
+            <ul>
+                <li>
+                    <img src="https://www.cislm.org/wp-content/uploads/2019/07/FI-Expanding-News-Desert_-1-300x300.png" width="200px" height="200px"><br>
+                    <figure-caption>December 11, 2017</figure-caption>
```

Comment:
What we display is a date and not a caption. So not an ideal tag.   But I like that you figured out that this tag exists.